modules/sb-install-service-catalog.adoc

```diff
@@ -10,47 +10,38 @@ enable it using the procedure below.
 
 .Procedure
 
-. Create the service catalog API server.
-.. Create a file named `svcat-api-server.yaml` with the following contents.
+. Enable the service catalog API server.
+.. Use the following command to edit the service catalog API server resource.
 +
-[source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: ServiceCatalogAPIServer
-metadata:
-  name: cluster
-spec:
-  logLevel: "Normal"
-  managementState: Managed
+$ oc edit servicecatalogapiservers.operator.openshift.io cluster
 ----
-.. Use the following command to create the service catalog API server.
+.. Under `spec`, set the `managementState` field to `Managed`, as shown below:
 +
 ----
-$ oc create -f svcat-api-server.yaml
+spec:
+  logLevel: Normal
+  managementState: Managed
 ----
+.. Save the file to apply the changes.
 +
-The service catalog API server is now installed into the `kube-service-catalog`
-namespace.
+The service catalog API server is now enabled in the
```

Comment:
I'd stick with "is now installed insto".  I'm thrown off a bit by the "is now" wording as "now" could correlate to the time immediately following the edit command vs 4.0 compared to 3.11.  Perhaps something like this?
"This change will cause the cluster operator to install the service catalog API Server component.  Note that as of  {product-title} this component is installed into the `openshift-service-catalog-apiserver` namespace (previously it was installed into `kube-service-catalog`).

Reply:
I'll update the wording a bit, though I think the displaying what the namespace was in 3.x is better info for the release notes. I'll update the suggested draft I sent you for the release notes service catalog section to include this and I'll email you the updated draft.